packages/components/pages/f-checkout/src/tenants/es-ES.js

```diff
@@ -1,6 +1,318 @@
-export default {
+const messages = {
     locale: 'es-ES',
-    text: 'I am a VueCheckout Component (ES)',
 
-    buttonText: 'Hacer pedido'
+    labels: {
+        addressGroup: 'Dirección',
+        deliveryOrderMethod: 'Hora de entrega',
+        collectionOrderMethod: 'Hora de recogida',
+        dineinOrderMethod: 'Hora de comida'
+    },
+
+    formFields: {
+        dineIn: {
+            tableIdentifier: {
+                label: 'Número de mesa o nombre',
+                validationMessages: {
+                    required: 'Incluye tu nombre/número de escritorio'
```

Comment:
I know this will have come from a proper translation, but isn't "escritorio" more of a desk than a table?